menu/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "twig-menu",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "fs-extra": "^0.18.2",
```

Comment:
When you run `npm install --save pattern-importer`, pattern-importer should automatically show up in this list.